pkg/endpoint/api/cell.go

```diff
@@ -44,6 +47,9 @@ var Cell = cell.Module(
 	// has started and the deletion queue has been drained to unlock the
 	// delete queue and thus allow CNI plugin to proceed.
 	cell.Invoke(unlockAfterAPIServer),
+
+	// Fence for Endpoint API.
```

Comment:
nit: Endpoint API context is useful but I think this still a bit terse. Probably one of these makes the intent clearer?
```suggestion
	// Fence unblocks once the Endpoint deletion queue has been processed.
```

or 

```suggestion
	// Fence blocks until this package is ready to process Endpoint API requests.
```